RawEventProcessor.cpp

```diff
@@ -23,17 +23,8 @@
 #include "Interpret.h"
 #include "StringUtils.h"
 
-/*****************************************************************************
-* New record types are in 10000 range to avoid collision with existing codes.
-*
-* 14688 was chosen for aggregate process creation records, given similarity
-* to windows 4688 events.
-*
-* 11309 was chosen for fragmented EXECVE records, following use of 1309 for
-* native AUDIT_EXECVE.
-*
-******************************************************************************/
-
+// Character that seperates key in AUDIT_FILTERKEY field in rules
+#define KEY_SEP 0x01
```

Comment:
Out of interest, is there a specific reason for using SOH as the seperator? Did you consider null, as this might be better for anything not expecting these merged keys?

Reply:
This same value is used in libaudit. I've updated the comment to reflect this.